.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,78 @@
+name: Bug
+description: 🐛 Let us know about an unexpected error, a crash, or an incorrect behavior.
+type: 'Bug'
+labels:
+  - 'type: bug'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a bug report! 🐛
+
+  - type: textarea
+    attributes:
+      label: A note for the community
+      value: |
+        <!-- Please keep this note for the community -->
+        * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        * If you are interested in working on this issue or have submitted a pull request, please leave a comment.
+        <!-- Thank you for keeping this note for the community -->
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem
+      description: >
+        Please provide a clear and concise description of what the bug is,
+        including what currently happens and what you expected to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: config
+    attributes:
+      label: VRL Program
+      description: |
+        Paste the relevant parts of your VRL program.
+
+        !! If your program contains sensitive information please redact it. !!
+      render: text
+
+  - type: input
+    id: version
+    attributes:
+      label: VRL Version
+      description: |
+        What VRL version is affected by this bug?
+    validations:
+      required: true
+
+  - type: textarea
+    id: debug
+    attributes:
+      label: Debug Output
+      description: |
+        Please share VRL diagnostics here.
+
+        !! Debug output may contain sensitive information. Please review it before posting publicly. !!
+      render: text
+
+  - type: textarea
+    id: data
+    attributes:
+      label: Example
+      description: |
+        Please share a VRL playground program. You can do this by clicking the `Share Program` button in
+        https://playground.vrl.dev/ and share the link.
```

```suggestion
        https://playground.vrl.dev/ and share the URL.
```

It'd be nice if the VRL playground could automatically copy to clipboard when "Share Program" is clicked.